appengine-java11/bigtable/src/main/java/com/example/bigtable/Main.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.example.bigtable;
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
```

Comment:
Isn't there a newer Bigtable Java API Client Library that doesn't use HBase?  Won't that be easier to use?

Reply:
@sduskis FYI